pyonfx/utils.py

```diff
@@ -143,95 +142,109 @@ class FrameUtility:
     You can use it to iterate over the frames going from ``start_ms``
     to ``end_ms`` and perform operations easily over multiple frames.
 
-    BACKGROUND KNOWLEDGE:
-
-    When reproducing a video+sub in a player, a line is rendered on the current frame
-    if the current time of the player is in between the line's start and end time.
-
-    The frame's duration can be determined by the fps value.
-
-    Let's now walk through an example: fps = 20 -> frame_duration = 50 ms.
-
-    The player will then seek for frames at the following times: 0, 50, 100, 150, ...
-
-    To accomodate this fact, frames' start and end times are positioned as follows:
-    - Frame 0: 0-25
-    - Frame 1: 25-75
-    - Frame 2: 75-125
-    - Frame 3: 125-175
-    - ...
-
-    This way, the frames' mid time will always be one of the player's seek times.
-    The only exception is the first frame, since we can't have negative times.
+    Both Constant Frame Rate (CFR) and Variable Frame Rate (VFR) videos are supported.
```

Comment:
I would put this comment into the `Timestamps` documentation instead of here.

Reply:
Done